# Intrinsic resistance mechanisms against antimicrobials #

## What is intrinsic resistance? ##

Whilst especially _Klebsiella_ and _E. coli_ are very problematic given their ability to very successfully acquire resistance mechanisms (i.e. horizontally acquired, most often via plasmids), we have very little understanding of the relevance of intrinsic resistance mechanisms - this can be porin de-activation (to stop import of antimicrobials), efflux pump upregulation (to pump antimicrobials faster out of the cell) or upregulation of plasmids / resistance genes broadening the range of antimicrobials to be able to resist against.

## Current work ##

A key area of our research here is on beta-lactam-beta-lacatamase-inhibitor combinations, and how intrinsic resistance mechanisms against these are regulated as this is often a result of resistance gene upregulation or plasmid upregulation. We are currently [investigating TAZ resistance](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000789) in projects with [Thomas Edwards](https://www.lstmed.ac.uk/about/people/dr-thomas-edwards) and collaborators, including since recently running our own in-house short- and long-read sequencing! 

We are furthermore investigating how bacterial outer membrane proteins, in particular porins and efflux pumps, contribute to the success of key _Klebsiella_ lineages, which nicely links Eva's background on outer membrane protein family evolution (e.g. [adhesins](https://pubmed.ncbi.nlm.nih.gov/27190006/), [the Omp85 family](https://pubmed.ncbi.nlm.nih.gov/25101071/), [TAM](https://pubmed.ncbi.nlm.nih.gov/25994932/), [secretion systems](https://pubmed.ncbi.nlm.nih.gov/23326233/)) with the more recent focus on large-scale genome analyses.
